app/views/application/partnerGraduateProgrammes.scala.html

```diff
@@ -0,0 +1,39 @@
+@(form: Form[_root_.forms.PartnerGraduateProgrammesForm.Data])(implicit request: Request[_], flash: Flash, user: Option[CachedData], feedbackUrl: String)
+
+@import views.html.application.assistancedetails.{ hasDisability, needsSupportOnlineAssessments, needsSupportAtVenue }
+@import views.html.widgets.{ pageProgress, saveAndContinueButton, radioBoxHorizontal, inputCheckboxGroup }
+@import helpers.CSRFieldConstructor._
+
+@main_template(title = "Partner graduate programmes", pageForms = Seq(form)) {
+    @pageProgress(3, 8)
+    <div class="text">
+        <h1 class="heading-xlarge" id="assistanceDetailsHeading">Partner Graduate Programmes</h1>
+    </div>
+    <div class="text">
+        <p>If you're successfully accepted on to the Fast Stream, you can defer
+            your place to complete a programme with one of our partners.</p>
+        <p>You can find out more <a href="https://www.gov.uk/guidance/civil-service-fast-stream-deferred-entry-options" rel="external">about our partner programmes</a>.</p>
```

Comment:
This doesn't look correct. Elsewhere, this would open in a new tab/window not replace the faststream tab. Can you check with Henry?